src/styles/base/_typography.scss

```
@@ -0,0 +1,31 @@
+body {
+    font-family: "Alan Sans";
```

bo har fontek, hatman dabe `fallback` fontek har law family fonta dani. ka agar aw font'a ba har dalil load nabu, system amel la fonteki shabih baw istfada bka
```suggestion
    font-family: "Alan Sans", sans-serif;
```